apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-button/index.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import LaunchModal from '../launch-modal';
+import './styles.scss';
+
+const LaunchButton: React.FunctionComponent = () => {
+	const [ isLaunchModalVisible, setLaunchModalVisibility ] = React.useState( false );
+
+	const handleClick = () => {
+		setLaunchModalVisibility( ! isLaunchModalVisible );
+	};
+
+	const handleModalClose = () => {
+		setLaunchModalVisibility( false );
+	};
+
+	return (
+		<>
+			<Button
+				aria-expanded={ isLaunchModalVisible }
+				aria-pressed={ isLaunchModalVisible }
+				aria-haspopup="menu"
+				onClick={ handleClick }
+			>
+				{ __( 'Launch site', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Launch site' )` *ES Score: 13*